tests/test_buildozer.py

```diff
@@ -1,13 +1,59 @@
+import re
 import os
+import codecs
 import mock
 import unittest
 import buildozer as buildozer_module
 from buildozer import Buildozer
 from six import StringIO
+import tempfile
 
 
 class TestBuildozer(unittest.TestCase):
 
+    def setUp(self):
+        """
+        Creates a temporary spec file containing the content of the default.spec.
```

Comment:
note: I know this is not the best approach performance wise since it creates a different temp file for each tests. But well it automates per-test spec file isolation and I don't think it impacts unit tests performances for now anyway